mypy/type_visitor.py

```diff
@@ -232,6 +233,14 @@ def visit_overloaded(self, t: Overloaded) -> Type:
     def visit_type_type(self, t: TypeType) -> Type:
         return TypeType.make_normalized(t.item.accept(self), line=t.line, column=t.column)
 
+    @abstractmethod
+    def visit_type_alias_type(self, t: TypeAliasType) -> Type:
+        # This method doesn't have a default implementation for type translators,
+        # because type aliases are special: some information is contained in the
+        # TypeAlias node, and we normally don't generate new nodes. Every subclass
+        # must implement this depending on its semantics.
```

Comment:
No reasonable way around this?

Reply:
Unfortunately no. This is kind of a fundamental limitation. Technically we can put something moderately reasonable, but then there is a high chance we will forget to override this when needed in some visitors we will add in future.